model/map-jpa/src/main/resources/META-INF/authz/policy/jpa-authz-policy-changelog-1.xml

```diff
@@ -129,5 +127,6 @@ limitations under the License.
         <addUniqueConstraint tableName="kc_authz_policy_scope" columnNames="policy_id, scope_id" deferrable="true" initiallyDeferred="true" />
         <addUniqueConstraint tableName="kc_authz_policy_resource" columnNames="policy_id, resource_id" deferrable="true" initiallyDeferred="true" />
         <addUniqueConstraint tableName="kc_authz_policy_associated_policy" columnNames="policy_id, associated_policy_id" deferrable="true" initiallyDeferred="true" />
+        <addUniqueConstraint tableName="kc_authz_policy_config" columnNames="fk_root, name" deferrable="true" initiallyDeferred="true" />
```

Comment:
I noticed that one too while working on the user store (was looking into how the mapped columns were represented in the changelog and came across this file. It might be good to leave it to a separate PR, perhaps as part of the fix for https://github.com/keycloak/keycloak/issues/12340?

Reply:
I've added comment: https://github.com/keycloak/keycloak/issues/12340#issuecomment-1166611193